aiidalab_widgets_base/viewers.py

```diff
@@ -1,12 +1,15 @@
 """Jupyter viewers for AiiDA data objects."""
-from __future__ import print_function
```

Comment:
Maybe we should remove all py27 relicts in a separate patch?

Reply:
Yes, I wanted to do a separate PR just to get rid of all the python2 things. But I think it doesn't harm if I remove some of them here and there.

Reply:
So do you want me to revet this things back?

Reply:
My suggested approach would be to apply all of the Py27-removal-related changes on a separate branch, we can quickly merge that, and then rebase on master. That would automatically clean up this PR. I am happy to help with that if you like.

Reply:
Alright, thanks for the proposition. I will then rebase things on top of your changes.